app/pages/500.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import StaticLayout from "components/Layout/StaticLayout";
+import getConfig from "next/config";
+import Link from "next/link";
+import { Button } from "@button-inc/bcgov-theme";
+import { useRouter } from "next/router";
+import Alert from "@mui/material/Alert";
+
+const Custom500 = () => {
+  const supportEmail =
+    getConfig()?.publicRuntimeConfig.SUPPORT_EMAIL || "ggircs@gov.bc.ca";
+  console.log("supportEmail", supportEmail);
+  const mailtoLink = `mailto:${supportEmail}?subject=Reporting an error`;
+
+  const router = useRouter();
+
+  return (
+    <>
+      <StaticLayout>
+        <Alert severity="error" icon={false}>
+          <h3>Sorry, something went wrong</h3>
+          <p>
+            An unexpected error occurred. If this error persists, contact us at{" "}
+            <Link href={mailtoLink}>
+              <a className={"email"}>{supportEmail}</a>
+            </Link>{" "}
+            and tell us what happened. Meanwhile you can{" "}
+            <Link href={"javascript:;"}>
+              <a onClick={() => router.back()}>refresh this page</a>
+            </Link>{" "}
+            or try again later.
+          </p>
+          <p>
+            <Link href="/" passHref>
+              <Button>Return Home</Button>
+            </Link>
+          </p>
+        </Alert>
+      </StaticLayout>
+      <style jsx>{`
+        a.email {
+          font-weight: bold;
+        }
+      `}</style>
```

Comment:
Why does this project do styling like this vs. stylesheet?

Reply:
The styles generates by `styled-jsx` will be scoped at the component level, ensuring that they don't have side effects on other components. If we did want to have global styles, then a stylesheet or a `<style jsx global>` element would work